### ISSUE TITLE
Manual language inconsistancy: no mention of string "main output gain", but that's what the UI tooltip states

### ISSUE THREAD

**Issue by mxmilkiib** [mixxxdj/mixxx#12269]
### Bug Description

See forum post [here](https://mixxx.discourse.group/t/main-output-gain-default-position-is-unity-or-what/28241).

Relavent manual section [here](https://manual.mixxx.org/2.3/en_gb/chapters/user_interface#headphone-and-main-mix-controls).

**Comment by ronso0** [comment]
The manual says
> **Main Knob**
> The Main Gain Knob controls the gain applied to the Main output signal

So for congruence you prefer **Main output gain knob**?

**Comment by mxmilkiib** [comment]
Yes, I guess. I just searched the manual for what the tooltip said and there were no results, which is not good. I'm not invested in the ultimate wording.

**Comment by ronso0** [comment]
Yeah, the search can be improved.
https://github.com/mixxxdj/manual/issues/500
